Gemfile.lock

```diff
@@ -0,0 +1,19 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    docile (1.3.2)
+    json (2.2.0)
+    simplecov (0.17.1)
+      docile (~> 1.1)
+      json (>= 1.8, < 3)
+      simplecov-html (~> 0.10.0)
+    simplecov-html (0.10.2)
+
+PLATFORMS
+  ruby
+
+DEPENDENCIES
+  simplecov
+
+BUNDLED WITH
+   2.0.2
```

Comment:
I think we can simplify this Gemfile for now with just SimpleCov. Everything else is standard, and we can call it with fewer lines of code. We can always add things as we need them!